lib/core/rancher/step.sh

```diff
@@ -80,10 +82,9 @@ function ih::setup::core.rancher::install() {
     # Installation and configuration of Rancher Desktop
     for _ in 1 2 3; do
 
-        # Detect Rosetta
-        if [[ $(sysctl -n sysctl.proc_translated) -eq 1 ]]; then
-            # Rosetta  Active
-            arch -arm64 -c brew reinstall ih-rancher
+        # Check if we have a Mac M1 and the terminal is running  over x86
+        if [[ $(sysctl -n sysctl.proc_translated) -eq 1 ]] && [ $(arch) = "i386" ]; then
+            mbrew reinstall ih-rancher
```

Comment:
I don't think we can assume `mbrew` alias is available at this point, we should probably just use the real command.